types/rlp_unmarshal_storage.go

```diff
@@ -115,8 +115,8 @@ func (r *Receipt) UnmarshalStoreRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) err
 		return err
 	}
 
-	if len(elems) != 3 {
-		return fmt.Errorf("expected 3 elements")
+	if len(elems) != 4 {
+		return fmt.Errorf("expected 4 elements")
 	}
```

Comment:
Is it possible to parse old Receipt?

Reply:
We can have if condition for each item

Reply:
Fixed af7a1d2b5fac4f1755c8b5d764f36275387e2b1c, please check again